client/lxd_containers.go

```diff
@@ -207,8 +207,10 @@ func (r *ProtocolLXD) tryCreateContainer(req api.ContainersPost, urls []string)
 
 			rop.targetOp = op
 
-			for _, handler := range rop.handlers {
-				_, _ = rop.targetOp.AddHandler(handler)
+			if r.supportsAuthentication {
```

Comment:
Can you use `SupportsAuthentication` everywhere in `client` so that when we search for usage of this function we get a clear picture of the usage.